<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RotateXYZDialog</class>
 <widget class="QDialog" name="RotateXYZDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RotateXYZ</string>
  </property>
  <widget class="QLabel" name="RotateXYZL">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RotateXYZ</string>
   </property>
  </widget>
  <widget class="QLabel" name="RotateX">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RotateX:</string>
   </property>
  </widget>
  <widget class="QLabel" name="RotateY">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RotateY:</string>
   </property>
  </widget>
  <widget class="QLabel" name="RotateZ">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RotateZ:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RotateXLE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RotateYLE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RotateZLE">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkPB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelPB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
